SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -369,6 +369,10 @@ void USpatialReceiver::OnRemoveComponent(const Worker_RemoveComponentOp& Op)
 		}
 		else
 		{
+			if (bInCriticalSection)
+			{
+				PendingAddActors.Remove(Op.entity_id);
```

Comment:
I guess this would still fail if we had add-remove-add in the same critical section, but I think that's exceedingly rare / impossible?

Reply:
Oh, maybe it still works, since we process ops in the critical section in order, so the add would be processed after this point.

Reply:
But an add-remove-add should actually result in a add, right ?

Reply:
Yeah, I was thinking it would remove the future add, but no it can't.